src/Maker/AbstractMaker.php

```diff
@@ -54,4 +55,15 @@ protected function addDependencies(array $dependencies, ?string $message = null)
             $message
         );
     }
+
+    public function configureComposerDependencies(DependencyManager $dependencyManager): void
+    {
+        // @TODO - method here in abstract prevents BC with signature added to `MakerInterface::class`
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+        // @TODO - do we deprecate this method in favor of the one above. then remove in 2.x
+        // @TODO - still have plenty of work todo to determine if thats possible or a good idea...
+    }
```

Comment:
Not a fan of touching the abstraction - but it keeps BC in check w/ the interface change.